nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkCommand.java

```diff
@@ -63,7 +69,7 @@ public String getRepository() {
     }
 
     /**
-     * @param query the query used to get the document set concerned by the operation
+     * @param query the query used to get the document set concerned by the action
```

Comment:
Why not having a normal builder pattern that include a build() method ? How do you ensure that mandatory field like actionId or query are always set ?

Reply:
We though it will be heavier to have to call build() each time.
Validation is done in service, and currently we just require `repository`. I'll add others.